zookeeper-client/zookeeper-client-c/pom.xml

```diff
@@ -114,6 +114,21 @@
               </arguments>
             </configuration>
           </execution>
+          <execution>
+            <id>build-c-client</id>
```

Comment:
Did you take this change from master?

Reply:
Initially, yes.  But it did not work as-is, because `master` uses `<tasks>` and it seems the Maven plugins pulled by 3.5 are missing support for it—so I adapted it by cargo-culting (and later cleaning up) the surrounding stanzas.